lib/product.rb

```diff
@@ -0,0 +1,28 @@
+
+class Product
+attr_reader :category, :name, :unit_price, :hoarded
+
+  def initialize(category,name,unit_price,quantity)
+    @category = category
+    @name = name
+    @quantity = quantity
+    @unit_price = unit_price
+    @hoarded = false
+  end
+
+  def quantity #IP shows instance variable quantity stored as string
+    @quantity.to_i
+  end
```

Comment:
nice, this is one way to do this! It allows you to preserve the way in which it was passed in, but get the data type requested 👍